Resources/Prototypes/_SpaceCats/Entities/Clothing/Back/backpacks.yml

```
@@ -14,4 +14,13 @@
-    sprite: _SpaceCats/BridgeOfficer/BOBackpack.rsi
\ No newline at end of file
+    sprite: _SpaceCats/BridgeOfficer/BOBackpack.rsi
+
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackSecurityEngineer
+  name: рюкзак полевого инженера
+  description: В два раза больше наручников!
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Clothing/Back/Backpacks/securityEngineer.rsi
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)